tests/examples/pacer/dockets/fake-rss/dcd.html

```diff
@@ -0,0 +1,8 @@
+<h3>THIS IS A FAKED UP DOCKET VIA RSS THROUGH recapupload.py<br>
```

Comment:
Can we tweak this since recapupload.py is in another project altogether? Maybe explain in here more about big cases bot?

Reply:
Well, are we testing what juriscraper receives, or are we testing what we would like it to receive? Because that's what it sends.

Do we change the comments in PACER HTML to remove references to random PACER `.js` and `.pl` files?

Reply:
Sometimes I do put comments into the other HTML files, yeah, if it clarifies what's up with them. Maybe just change the `h3` to say something like, "THIS IS A FAKED UP DOCKET VIA RSS MADE BY THE BIG CASES BOT". I just see somebody getting totally stumped where the recapupload.py file is.

Reply:
Done.